veps/sig-compute/decentralized-migration/decentralized-migration-vep.md

```diff
@@ -0,0 +1,279 @@
+# VEP #Decentralized live migration
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+Proposal to decentralize live migration into several flows to allow for more flexible live migration. This includes live migration to a different VM in a different namespace or cluster.
+
+## Motivation
+
+Current live migration is limited in its flexiblity because the migration is centralized. This limits live migration to the same namespace and cluster. There are use cases where it would be beneficial to live migrate between namespaces and even between clusters.
+
+In order to support those use cases, it is needed that live migration is decentralizated into separate flows:
+
+- synchronization
+- source
+- target
+
+The synchronization flow establishes a connection between the source and target and synchronizes the `VirtualMachineInstance` status between the two. 
+
+The target flow sets up the receiving `virt-launcher` pod, and notifies the source entity through the `VirtualMachineInstance` status, that the migration is ready to begin.
+
+The source flow waits for the target flow to be ready and initiates the migration. Before the migration begins it generates the migration parameters which include the name of the target and any translation of fields/path/names to the target.
+
+## Goals
+
+ - Break the `VirtualMachineInstanceMigration` into two entities. With a separate flow for each entity. A `source` and a `target` flow. 
```

Comment:
It's still one CR, right? Thus also one entity. But it's api is extended in order to decouple send from receive?

Reply:
Correct, but how the controller handles the entities will be two completely separate flows. They can be processed completely independent of each other and in parallel if needed. The flow right now is very intertwined.

Reply:
I reworded it a bit, I like the decoupling verbage.